recipes/mvapich2/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -ex
+
+unset F77 F90
+
+export FFLAGS="-fallow-argument-mismatch ${FFLAGS}"
+
+export CC=$(basename "$CC")
+export CXX=$(basename "$CXX")
+export FC=$(basename "$FC")
+
+build_with_rdma=""
+if [[ "$target_platform" == linux-* ]]; then
+    echo "Build with RDMA support"
+    build_with_rdma="--with-rdma=$PREFIX --enable-rdma-cm "
+fi
+
+./configure --prefix=$PREFIX \
+            --enable-fast=none \
+            --enable-g=all \
```

Comment:
probably don't want to disable optimization and force debug builds?

```suggestion
```

Reply:
This first version is meant for testing and debugging purposes, so I would like to disable all optimization and leave the debug option on

Reply:
Makes sense!